packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/calibrate-operation.ts

```diff
@@ -60,7 +64,9 @@ export const CalibrationOperationCiemss: Operation = {
 			errorMessage: { name: '', value: '', traceback: '' },
 			numIterations: 100,
 			numSamples: 100,
-			endTime: 100
+			endTime: 100,
+			stepSize: 1,
+			learningRate: 0.1
```

Comment:
0.03 is the default that the pyciemss team uses. We should probably be consistent here as we often ignore the additional settings sections when testing and then get questions about why our results differ

Reply:
thanks!